docs/installation/SUSE.md

```diff
@@ -8,31 +8,48 @@ parent = "smn_linux"
 +++
 <![end-metadata]-->
 
-# openSUSE
+# openSUSE and SUSE Linux Enterprise
 
-Docker is available in **openSUSE 12.3 and later**. Please note that due
-to its current limitations Docker is able to run only **64 bit** architecture.
+This page provides instructions for installing and configuring Docker on openSUSE
+and SUSE systems.
 
-Docker is not part of the official repositories of openSUSE 12.3 and
-openSUSE 13.1. Hence  it is necessary to add the [Virtualization
-repository](https://build.opensuse.org/project/show/Virtualization) from
-[OBS](https://build.opensuse.org/) to install the `docker` package.
+## Prerequisites
 
-Execute one of the following commands to add the Virtualization repository:
+Due to its current limitations Docker is able to run only **64 bit** architecture.
 
-    # openSUSE 12.3
-    $ sudo zypper ar -f http://download.opensuse.org/repositories/Virtualization/openSUSE_12.3/ Virtualization
+### openSUSE
 
-    # openSUSE 13.1
-    $ sudo zypper ar -f http://download.opensuse.org/repositories/Virtualization/openSUSE_13.1/ Virtualization
+Docker is part of the official openSUSE repositories starting from 13.2. No additional repository
+is required on your system.
 
-No extra repository is required for openSUSE 13.2 and later.
+Bleeding edge version of Docker can be found inside of the repositories maintained
+by the [Virtualization:containers project](https://build.opensuse.org/project/show/Virtualization:containers)
+on the [Open Build Service](https://build.opensuse.org/).
+This project delivers also other packages that are related with the Docker ecosystem (e.g. docker-compose).
 
-# SUSE Linux Enterprise
+#### SUSE Linux Enterprise
 
-Docker is available in **SUSE Linux Enterprise 12 and later**. Please note that
-due to its current limitations Docker is able to run only on **64 bit**
-architecture.
+Docker is officially supported on **SUSE Linux Enterprise 12 and later**.
+
+The latest supported Docker packages can be found inside of the `Container`
+module.
+
+Enabling the Container Module:
+
+ * Start YaST, and select *Software > Software Repositories*.
```

Comment:
Can these steps also be performed via the command-line? The steps below "installation" take place on the command-line, would be nice to show how to do these steps via the command-line as well